src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -46,6 +44,22 @@ public void Update()
         {
             DataContext = _settings;
             SetSelectedItem();
+
+            UpdateWorkArea();
+        }
+
+        public void UpdateWorkArea()
+        {
+            Rect workArea = Settings.WorkArea;
+
+            if (workArea.Height < 900 || MonitorViewModel.IsDesktopsPanelVisible)
```

Comment:
Since you are touching this code, let's replace `900` and the `180`  below with constants.

Reply:
Done.